packages/components/src/guide/style.scss

```diff
@@ -92,12 +87,15 @@
 	height: 80vh;
 	max-height: 575px;
 
-	@media (max-width: $break-small) {
-		bottom: 5%;
-		left: $grid-unit-20;
-		right: $grid-unit-20;
-		top: 5%;
-		margin: 0 auto;
+	@media ( max-width: $break-small ) {
+		margin: auto;
+		min-width: 280px;
```

Comment:
After testing a bit more I'm favoring omitting the change to `min-width`. The component fits at 320px with a little room on each side and even down at 280px the "next" and "get started" buttons are still accessible. I think the reduced min-width is a good but seems non-crucial and better in a separate PR which also styles the images to resize instead of just hiding their overflow.
```suggestion
```